client/src/App.js

```diff
@@ -20,6 +24,26 @@ export const PATHS = {
   register: "/register",
 }
 
+/*
+  STORE SETUP
+*/
+
+const mapStateToProps = state => {
```

Comment:
The state should correspond to how `combineReducers` combines reducers, e.g. `countState` is the key that stores the store state change brought about by the `incrementReducer`